### ISSUE TITLE
Breder gebruik van AggregateId

### ISSUE THREAD

**Issue by kad-bloemy** [kadaster-labs/sensrnet-home#185]
Naar aanleiding van een gesprek met Beryl Dreijer en Anne-Maartje Duqué:
Zij vinden AggregateId een zeer interessante groepering van apparaten en sensoren. In Amsterdam worden dit soort groeperingen gebruikt voor bijvoorbeeld sluisbewaking: sensoren die eigenlijk een ander doel hebben kunnen ook worden ingezet voor de bediening en monitoring van sluizen. Zij zouden daar eigenlijk een superentiteit van willen zien met ook een zoekfunctie erop.

**Comment by marcvanandel** [comment]
Is dat een breder gebruik van AggregateId of een 'super entiteit' met een eigen ID? Ik vermoed eigenlijk het laatste. Een AggregateId is de 'technische sleutel' naar een `Device`. Als meerdere devices bij elkaar een 'nieuwe groep' vormen, een gezamenlijke eenheid vormen, dan is dat een nieuw concept, een nieuwe entiteit ... met een eigen ID.

Overigens begrijp ik deze wens wel en vind ik deze ook zeer relevant. Ik denk dat het vastleggen van de metadata van sensoren (`Device`s in het sensorenregister) de basis is van vele contextuele 'lagen' en entiteiten die daar gebruik van maken. Het begrip 'sluis' is daar een voorbeeld van. Persoonsgegevensverwerking is daar ook een voorbeeld van. Verkeerssystemen zijn daar een voorbeeld van. Luchtkwaliteit is daar een voorbeeld van. En ga zo maar door. De vraag is wat er nu **ín** het sensorenregister hoort of wat daar buiten hoort en wel _gebruik maakt van_ het sensorenregister... 🤔

**Comment by kad-bloemy** [comment]
Device is eigenlijk ook al een groepering, maar dan op basis van de locatie en het "omhulsel", die diverse sensoren kan bevatten. Deze is zichtbaar voor de burger. Aggregate is dan meer een logische of functionele groepering. Ik kan mij voorstellen dat bijvoorbeeld een Rijkskwaterstaat daar ook wel oren naar heeft voor aan elkaar geknoopte verkeerssensoren zoals in trajectcontroles. Voor de burger is deze informatie best relevant en kan zeker ook meerwaarde hebben. Voer voor de FAB?

**Comment by marcvanandel** [comment]
Ja, dit is zeker voer voor FAB. Ik denk wel - en nog steeds - dat AggregateId niet het juiste 'ding' is, maar dat we nog iets missen. In de datamodel voorbeelden heb ik de CityBeacon als voorbeeld gebruikt van een topografisch object dat hetzelfde is voor meerdere devices. Zouden er meerdere 'tags' of zo toe te voegen moeten zijn om aan te geven dat meerdere devices bij elkaar horen?
